tests/plugins/imagebase/features/manual/upload.html

```diff
@@ -3,78 +3,76 @@
 	<p></p>
 </div>
 
-<script src="../../../easyimage/manual/_helpers/tools.js"></script>
 <script>
-	if ( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) {
+	if ( isUnsupportedEnvironment() ) {
 		bender.ignore();
```

Comment:
Won't it be easier to wrap everything in IIEFE and `return` after `bender.ignore`?

Reply:
Now regular condition will work. I changed it before because I haven't found the source of the issue.
Right now `else` statement is unnecessary.
Problem was here, where word `default` doesn't work with dot notation in IE and has to be transformed int bracket one, what breaks entire script tag and prevent of ignoring the test.
https://github.com/ckeditor/ckeditor-dev/blob/47dcbec76d985a8359aab4c15a321c6c89fec245/tests/plugins/imagebase/features/manual/upload.html#L58